ApplyToTheIndustry/Assets/Scripts/AI Grader/Grader.cs

```diff
@@ -81,25 +81,26 @@ public void CalculateTotalPoints()
         }
     }
 
-    public void GetFeedback()
+    public string GetFeedback()
     {
         if (Totalpoints < 15)
```

Comment:
We could put these hard-coded cutoff points in a scriptable object so that they can be balanced or changed according to the specific position needs.